docs/concepts/resource-manager.md

```diff
@@ -119,7 +119,7 @@ A ManagedResource has an optional `.spec.class` field that allows to indicate th
 
 #### Conditions
 
-A ManagedResource has a ManagedResourceStatus, which has an array of ManagedResourceConditions. ManagedResourceConditions currently include:
+A ManagedResource has a ManagedResourceStatus, which has an array of Conditions. Conditions currently include:
```

Comment:
BTW: shall we now generate API docs for `pkg/apis/resources` too?

Reply:
Good idea, will do.